notebooks/ai-ml/HuggingFaceEmbeddingGenerationInSpark.ipynb

```diff
@@ -0,0 +1,772 @@
+{
```

Comment:
<p>Add a cell above this: "Create a Spark Session and load the Spark package <code>spark-avro</code>. </p><p> </p><p> </p><p> </p><p>Nit: formatting:</p><p> </p><p><code>spark = SparkSession.builder.appName("Embeddings")\</code></p><p><code>  .config("spark.jars.packages", "org.apache.spark:spark-avro_2.13:3.5.1") \</code></p><p><code>  .getOrCreate()</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudDataproc/cloud-dataproc/pull/171/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>